docs/capabilities/portfolio-objectives.md

```diff
@@ -0,0 +1,75 @@
+
+**Portfolio Objective**:  A type of objective that targets a group of systems based on similar metadata, aiming to provide an efficient and user-friendly experience that removes the need to set objectives individually for each system and better prioritization. (General rule) 
+
+
+<img src="../images/portfolio-objectives-main-table.png" width="700" />
+
+
+**System Objective**: A type of objective specific to an individual system within a portfolio that does not follow the general rule. When an objective is set at the system level, the system no longer follows the objective defined at the portfolio level.
+
+
+<img src="../images/portfolio-objectives-system-detail.png" width="700" />
+
+
+## How to define portfolio objectives? 
+
+To define a portfolio objective, the user can click on the "Add Portfolio Objective" button on the top right of the screen:
+
+<img src="../images/portfolio-objectives-add-arrow.png" width="700" />
+
+Upon clicking the button, a pop-up will appear to guide you during the creation of a portfolio objective.
+
+The pop-up will guide you toward configuring the portfolio objective:
+
+<img src="../images/portfolio-objectives-add-popup.png" width="700" />
+
+There will be several dropdown menus available in the pop-up.
+
+
+1. There will be a Capability dropdown, that allows the user to select the specific capability for which the portfolio objective will be defined.
+
+2. Then, the Type dropdown, allows the user to select a given objective type that is part of the capability selected in the previous step. In the example, we see that the Test Code Ratio is set as a type. This is due to the fact that the "Test Code Ratio" objective is a type of objective that belongs to the Maintainability capability.
+
+3. Finally, the last step allows for defining the actual value that we want our portfolio objective to be. 
+This final field can be of two distinct types: either a free-text field, in the case of numeric objectives, or a dropdown containing a series of pre-defined values, that will be the allowed values for the given objective type.
+
+Note also the indicator that the objective will apply to all systems in a given portfolio.
+
+4. Finally, we have a free-text field called Rationale. This field can be used to add any additional details on why a given objective was set and add extra details regarding the values for ensuring full clarity to all involved stakeholders.
+
+
+## Importance of the Rationale field
+
+One of the main goals of portfolio objectives is that they allow our customers to codify business objectives in Sigrid. 
+A portfolio objective is not, exclusively, a technical objective, but, it’s also a business objective for our customers’ portfolio. 
```

Comment:
I would add something about communication, since that's effectively what a rationale is. And it works both ways: Connecting technical objectives to business goals help to communicate to management *why* something is important. And reversely, pointing to business goals makes it more explicit to development teams what the eventual goal of something is.

Reply:
Added a bit more of an explanation here